windows-mixin/jsonnetfile.json

```diff
@@ -4,12 +4,21 @@
     {
       "source": {
         "git": {
-          "remote": "https://github.com/grafana/grafonnet-lib.git",
-          "subdir": "grafonnet"
+          "remote": "https://github.com/grafana/jsonnet-libs.git",
+          "subdir": "windows-observ-lib"
         }
       },
-      "version": "master"
+      "version": "vzhuravlev/jl-windows-new"
```

Comment:
You'll want to have a follow-up PR to change this to master, or perhaps you can reference the specific commit SHA. I believe the branch usually gets deleted on merge.